storybook/src/Ali/Molecules/Privacy/Privacy.stories.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Privacy from "./Privacy";
+
+export default {
+  title: "Tramsi/Molecules/Privacy/Privacy",
+  component: Privacy,
+};
```

Comment:
1. Use other control instead of text 

I suggest read https://storybook.js.org/docs/react/essentials/controls/